Bakis/ClientApp/src/app/components/map/map.component.ts

```diff
@@ -122,28 +126,44 @@ export class MapComponent implements AfterViewInit  {
     this.sightsMarkers = tempMarkers;
   }
 
-  getGameDistance() {
-    if (this.sightsMarkers.length == 0) {
-      this.sendReceiveService.sightReceive$.subscribe((data) => {
-        this.setPopUp(data);
-      });
+  getGameDistance(isSight) {
+    if (isSight) {
+      if (this.sightsMarkers.length == 0) {
+        this.sendReceiveService.sightReceive$.subscribe((data) => {
+          this.setPopUp(data, isSight);
+        });
+      }
+      else {
+        this.setPopUp(this.sightsMarkers, isSight);
+      }
     }
     else {
```

Comment:
Would it make more sense to remove this else so it goes through sight and not sight?